client/app/components/NavBar/index.js

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router";
 import classNames from "classnames";
+import ContactModal from "../Modal";
```

Comment:
non-child components should be referenced as `from "Components/<ComponentName;"`